pkgs/by-name/ha/harlequin/package.nix

```
@@ -2,14 +2,31 @@
   lib,
   python3Packages,
   fetchFromGitHub,
-  harlequin,
-  testers,
   nix-update-script,
   versionCheckHook,
   glibcLocales,
   withPostgresAdapter ? true,
   withBigQueryAdapter ? true,
 }:
+let
+  # Temporarily use textual 0.82 until harlequin is updated for 0.86
+  # https://github.com/NixOS/nixpkgs/issues/364867
+  textual_0_82 = (
+    python3Packages.textual.overrideAttrs (old: rec {
+      version = "0.82.0";
+
+      src = fetchFromGitHub {
+        owner = "Textualize";
+        repo = "textual";
+        rev = "refs/tags/v${version}";
+        hash = "sha256-belpoXQ+CkTchK+FjI/Ur8v4cNgzX39xLdNfPCwaU6E=";
+      };
+      disabledTests = old.disabledTests ++ [
+        "test_selection"
+      ];
+    })
+  );
```

We can remove the parenthesis here:

```suggestion
  textual_0_82 = python3Packages.textual.overrideAttrs (old: rec {
    version = "0.82.0";
    src = fetchFromGitHub {
      owner = "Textualize";
      repo = "textual";
      rev = "refs/tags/v${version}";
      hash = "sha256-belpoXQ+CkTchK+FjI/Ur8v4cNgzX39xLdNfPCwaU6E=";
    };
    disabledTests = old.disabledTests ++ [
      "test_selection"
    ];
  });
```